### ISSUE TITLE
local build of 1.6.0 fails

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#265]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/265) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

While a build of stable works we run in errors on development branch

```
#!text

RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/pytest-runner/r
Traceback (most recent call last):
  File "/home/mss/miniconda2/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 387, in main
    execute(sys.argv[1:])
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 378, in execute
    noverify=args.no_verify)
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/api.py", line 184, in build
    need_source_download=need_source_download, config=config, variants=variants)
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/build.py", line 1733, in build_tree
    notest=notest,
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/build.py", line 1125, in build
    utils.check_call_env(cmd, env=env, cwd=src_dir)
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/utils.py", line 678, in check_call_env
    return _func_defaulting_env_to_os_environ(subprocess.check_call, *popenargs, **kwargs)
  File "/home/mss/miniconda2/lib/python2.7/site-packages/conda_build/utils.py", line 674, in _func_defaulting_env_to_os_environ
    return func(_args, **kwargs)
  File "/home/mss/miniconda2/lib/python2.7/subprocess.py", line 186, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/bash', '-x', '-e', '/home/mss/miniconda2/conda-bld/mss_1503394769301/work/conda_build.sh']' returned non-zero exit status 1

```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

looks like we now need on build 

pytest-runner

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We have introduced this by 655b264a0dbee1c7c161e4eabca5eb81b02d6f91

@joernu76 I think we should move pytest-runner to requirements.d/development.txt

Not moving it means we have to add it to meta-yaml too and  users got it installed.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

partly reverted 655b264, fixes #265

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #460)

partly reverted 655b264, fixes #265
